spec/pushmi_pullyu/preservation_queue_spec.rb

```diff
@@ -4,15 +4,18 @@
 RSpec.describe PushmiPullyu::PreservationQueue do
   describe 'a queue with 3 items in it' do
     let(:queue) { PushmiPullyu::PreservationQueue.new(poll_interval: 0, queue_name: 'test:pmpy_queue') }
+    let!(:redis) { Redis.new }
```

Comment:
Doesn't it yell at you about not using `let` variables in before blocks? That's why I avoided it.